dart/lib/src/exception_cause.dart

```diff
@@ -0,0 +1,6 @@
+class ExceptionCause {
```

Comment:
```suggestion
abstract class ExceptionCause {
```

If the goal is to be an interface, we can make `abstract`.

Reply:
This is just a 'Tuple' to hold both/either exception & stackTrace. Not sure about the name though.

Reply:
That was for `ExceptionCauseExtractor`, made the comment in the wrong place.

Reply:
See https://github.com/getsentry/sentry-dart/pull/1198/files#r1071406275